README.md

```diff
@@ -61,7 +61,7 @@ CREATE TABLE error_counts AS
 
 
 # Documentation
-You can [find the KSQL documentation here](https://github.com/confluentinc/ksql/tree/v0.3/docs#ksql-documentation)
+You can [find the KSQL documentation here](https://github.com/confluentinc/ksql/tree/v0.4/docs#ksql-documentation)
```

Comment:
Can you update the references to the demo docs on lines 14/15?

When I did this the first time I made all these links references and grouped the definitions together at the end so it would be harder to miss one. Maybe we should go back to that.

Reply:
done.